local-cli/link/pods/addPodEntry.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+module.exports = function addPodEntry(podLines, linesToAddEntry, podName, nodePath) {
+	const newEntry = `pod '${podName}', :path => '../node_modules/${nodePath}'\n`;
+
+	if (!linesToAddEntry) {
+		return;
```

Comment:
This file has wrong indentation. Try running `prettier` for all the files in your PR (not sure if you have it integrated already)